cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/replication/ReplicationListener.java

```diff
@@ -0,0 +1,75 @@
+package com.cloudant.sync.internal.replication;
+
+import com.cloudant.sync.event.Subscribe;
+import com.cloudant.sync.event.notifications.ReplicationCompleted;
+import com.cloudant.sync.event.notifications.ReplicationErrored;
+import com.cloudant.sync.replication.ReplicationPolicyManager;
+import com.cloudant.sync.replication.Replicator;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class is not intended as API, it is public for EventBus access only.
+ * API consumers should not call these methods. See #setReplicationsCompletedListener for more
```

Comment:
Fixed in d06de6bd6c02a55e24d09d038c08bf5388376be1 with some description about what the class is actually for.